packages/flame_markdown/lib/custom_attribute_syntax.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:markdown/markdown.dart';
+
+// NOTE: values obtained from file `charcode.dart` from the markdown package
```

Comment:
sadly we cannot access the file as it is not exported. tbh I do prefer my naming schema though :P